schema/src/main/java/org/hl7/v3/RCMRMT030101UKAgentDirectory.java

```diff
@@ -0,0 +1,22 @@
+package org.hl7.v3;
```

Comment:
Not a blocker, but I'd suggest that these interfaces are more strongly assoccited with the PSS package, as that's what cares about their implementation.

Reply:
ok, I think it is a good call, I will consider it in some later PRs.